trunner/testcase.py

```diff
@@ -192,6 +199,7 @@ def __init__(
     ):
         super().__init__(name, target, timeout, exec_cmd, use_sysexec, status)
         self.load_module(harness_path)
+        self.type = 'harness'
```

Comment:
**if** we would ever want to clearly specify the test type, we would like to use some enumerated type - not a string constant (which can be easily mistyped)

Reply:
solved with `is_type` logic removed